main.css

```diff
@@ -45,3 +45,40 @@ a {
 a:hover {
         color: #9A0008
 }
+/******************************
+footer
+******************************/
+footer
+{
+        padding-bottom:10px;
+        color: white;
+}
+footer p,h3,h4,h2
```

Comment:
This will affect *all* `h2`, `h3`, and `h4` tags instead of just the ones in a `footer` tag.

Reply:
Thanks for correcting me. :D